devdoc/constbuffer_requirements.md

```diff
@@ -362,48 +362,82 @@ CONSTBUFFER_FROM_BUFFER_RESULT CONSTBUFFER_from_buffer(const unsigned char* sour
 
 **SRS_CONSTBUFFER_02_073: [** If there are any failures then shall fail and return `CONSTBUFFER_FROM_BUFFER_RESULT_ERROR`. **]**
 
+### CONSTBUFFER_createWritableHandle
 
-### CONSTBUFFER_create_writable_handle
+```c
+ CONSTBUFFER_WRITABLE_HANDLE CONSTBUFFER_createWritableHandle (uint32_t size)
+```
+
+`CONSTBUFFER_createWritableHandle` construct a new `CONSTBUFFER_WRITABLE_HANDLE` of size `size`. Buffers of `CONSTBUFFER_WRITABLE_HANDLE` can be overridden (the same buffer can be reused to fill).
+
+**SRS_CONSTBUFFER_51_001: [** If `size` is 0, then CONSTBUFFER_createWritableHandle shall fail and return NULL. **]**
```

Comment:
>CONSTBUFFER_createWritableHandle [](http://example.com/codeflow?start=51&length=32)

We put backticks (') around the function names in the .md so they look nicer in the rendered md.

Reply:
Added.